man/coredumpctl.xml

```diff
@@ -217,6 +218,13 @@
         will be used. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--debugger-args=</option><replaceable>ARGS</replaceable></term>
+
+        <listitem><para>Pass the given <replaceable>ARGS</replaceable> as command line arguments
+        to the debugger.</para></listitem>
```

Comment:
Maybe suggest that people should enclose the passed argument in `''` to ensure the arguments are handled by gdb and not by coredumpctl